### PR TITLE
evaluator-rules: Improve the disclaimer a bit

### PR DESCRIPTION
**Pull request by fviernau**
Drop the copy pasted mention of "ORT examples", make the disclaimer a bit more prominent and avoid using the term example overall while still making clear that no legal recommendation is made. While at it wrap it at 80 chars.

### CODE REVIEW

File: evaluator-rules/src/main/resources/evaluator.rules.kts [hunk 1]
```diff
@@ -17,13 +17,11 @@
...
-/*******************************************************
- * Example OSS Review Toolkit (ORT) .rules.kts file    *
- *                                                     *
- * Note this file only contains example how to write   *
- * rules. It's recommended you consult your own legal  *
- * when writing your own rules.                        *
- *******************************************************/
+/******************************************************************************
+ * DISCLAIMER: This file only illustrates how to write evaluator rules, it's  *
+ *             no recommendation in any way. It is recommended to consult     *
```
Comment (tsteenbe) [inline evaluator-rules/src/main/resources/evaluator.rules.kts:22]: It is recommended -> It's recommended

**Review by tsteenbe — CHANGES_REQUESTED** [review]

File: evaluator-rules/src/main/resources/evaluator.rules.kts (hunk 1 above)
Comment (mnonnenmacher) [reply evaluator-rules/src/main/resources/evaluator.rules.kts]: I think in formal texts "it is" is the better choice than using a contraction.

Comment (fviernau) [reply evaluator-rules/src/main/resources/evaluator.rules.kts]: @tsteenbe ok with changing it back again?

Comment (fviernau) [reply evaluator-rules/src/main/resources/evaluator.rules.kts:22]: Changed it anyway as it's minor.

**Review by tsteenbe — APPROVED** [review]

**Pull request merged**